web/modules/custom/rdf_schema_field_validation/rdf_schema_field_validation.install

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/**
+ * @file
+ * Contains install and update hooks of the rdf_schema_field_validation module.
+ */
+
+use Drupal\rdf_entity\Entity\RdfEntityMapping;
+use EasyRdf\Graph;
+use EasyRdf\GraphStore;
+
+/**
+ * Implements hook_update_N().
+ */
+function rdf_schema_field_validation_update_8102() {
+  $graph_uri = 'http://adms-definition';
+  $class_definition = 'http://www.w3.org/2000/01/rdf-schema#Class';
+
+  $sparql_endpoint = \Drupal::service('sparql_endpoint');
+  $connection_options = $sparql_endpoint->getConnectionOptions();
+  $connect_string = 'http://' . $connection_options['host'] . ':' . $connection_options['port'] . '/sparql-graph-crud';
+  // Use a local SPARQL 1.1 Graph Store.
+  $gs = new GraphStore($connect_string);
+  $graph = new Graph();
+  try {
+    $graph->parseFile(DRUPAL_ROOT . '/../resources/fixtures/adms-definition.rdf');
+  }
+  catch (\Exception $e) {
+    throw $e;
+  }
+  $out = $gs->replace($graph, $graph_uri);
+  if (!$out->isSuccessful()) {
```

Comment:
This is useless. GraphStore::replace() is already throwing an error if isSuccessful() returns FALSE. See https://github.com/njh/easyrdf/blob/master/lib/GraphStore.php#L153